tests/features/common/kie-kieserver-common.feature

```diff
@@ -95,46 +95,62 @@ Feature: Kie Server common features
   Scenario: Check custom users are properly configured
     When container is started with env
       | variable                   | value         |
-      | KIE_ADMIN_USER             | customExe     |
-      | KIE_ADMIN_PWD              | custom" Exe!0 |
-    Then file /opt/eap/standalone/configuration/application-users.properties should contain customExe=a4d41e50a4ae17a50c1ceabe21e41a80
-     And file /opt/eap/standalone/configuration/application-roles.properties should contain customExe=kie-server,rest-all,admin,kiemgmt,Administrators
+      | KIE_ADMIN_USER             | customAdm     |
+      | KIE_ADMIN_PWD              | custom" Adm!0 |
+    Then file /opt/eap/standalone/configuration/application-users.properties should contain customAdm=a4d41e50a4ae17a50c1ceabe21e41a80
```

Comment:
As we change user name and password, will this check pass? Is the hash still same?